rules/game_process.tex

```diff
@@ -124,8 +124,41 @@ \subsubsection{Invalid Goal}
 In these cases a goal is not scored (that is, the goal is ruled invalid), and the game will proceed with a Goal Kick (\cf \cref{sec:free_kick}).
 The head referee should also advise why the goal is invalid, such as by calling ``Not indirect'', ``Played with hands'' or ``No own goal''.
 
-\subsubsection{Indirect Kick}
-\label{sec:indirect_kick}
+\subsubsection{Indirect Kick - Champion Cup}
+\label{sec:indirect_kick_champion}
+
+A team may only score a goal by making a second contact with the ball within is own team from any affected start or restart into \texttt{playing}. 
+
+\begin{itemize}
+  \item A robot may not score a goal from a direct kick, including via deflections.
+  \item Kick considered to be indirect are \texttt{Kick-off}, \texttt{Kick-in}, \texttt{Pushing free kick} and \texttt{Penalty kick}.
+  \item The ball must be deliberately played-at a second time by another robot of its own team before a goal may be scored.
+    A deliberate play at the ball includes successfully kicking the ball, dribbling the ball (and subsequently leaving possession of the ball), or the goalkeeper playing at the ball with its hands.
+  \item If an indirect free kick is kicked directly into the opponent's goal, the goal does not count and a goal kick is awarded.
+  \item Note that an own-goal may always be scored without requiring an indirect kick.
+  \item Direct kick is allowed for the duration of the penalty kick, but this rule still applied after this kick. Contact in the penalty kick count among this rule.  
+\end{itemize}
+
+\begin{description}
+  \item[Example 1:] Player 2 (of the red team) kicks the ball to Player~3 (of the red team), who then kicks the ball into the blue-team's goal.
+  This is a successful indirect kick, and the goal counts.
+  \item[Example 2:] Player 2 (of the red team) kicks the ball at the goal, and it is deflected off the side of the foot of a blue-team's robot into the goal.
+    This is \textit{not} an indirect kick, the goal does not count and a goal kick is awarded to the blue team.
+  \item[Example 3:] Player 2 (of the red team) kicks the ball ``upfield''.
+    A blue-team robot kicks the ball a short distance, after which Player~2 kicks the ball again into the blue team's goal.
+    This is \textit{not} an indirect kick, the goal does not count and a goal kick is awarded to the blue team.
+  \item[Example 4:] Player 2 (of the red team), walks up to and dribbles the ball.
+    Player~2 then stop, and visibly back away from the ball, before approaching to dribble a \textit{second} time.
+    The robot then scores. This is \textit{not} a successful indirect kick and a goal kick is awarded.
+  \item[Example 5:] Player 2 (of the red team) kicks the ball ``upfield''.
+  A blue-team robot kicks the ball again into the red team's goal.
+  In this case, the blue teams does not need a indirect kick and the goal is counts.
```

Comment:
This example seems wrong???

Reply:
My understanding of indirect kick is if the team don't do the kick, they don't need a indirect. While making indirect kick for both may make more dynamics, it's a step away from the fifa rule. Let me know if I'm still wrong.

Reply:
In my understanding, both teams need an indirect kick and this example is wrong. Also I agree that it might not fully align with the fifa rules I would say that this would again be a special case and makes it harder for the referee. So for this year I would say we always require from both teams to do an indirect kick.

Reply:
Why do you think that both teams need (or specifically that the Blue team needs) an indirect kick for example 5. I wouldn't agree with both teams requiring an indirect kick. Before the kick was taken there was no requirement for the blue team to do an indirect kick and this should not change as a result of what the red player did.

Example 3 is also problematic. I would have thought that as soon as possession transferred to the team that was defending the kick, we're back to normal playing. I understand that you are probably trying to catch the case where the red player executing the indirect kick just kicks at the blue opponent and then immediately follows up the rebound to shoot and score. However, example 3 creates questions for referees. How long is "short kick" by the blue player and when can we consider that the indirect kick (for red) is over and we're back to normal play?

Reply:
In my understanding after each start or restart into playing each of the teams has to do the indirect kick in order to score a goal which means for each team 2 different robots of the team have to touch the ball. So in this example it's not stated who has touched the ball. If for both teams already two different robots have touched the ball then this example holds. Otherwise not.

Reply:
I don't think that is obvious is in the rules and it is certainly not intuitive. Normally any rules about indirect free kicks or indirect kicks apply only to the team awarded the kick. If your interpretation is as you intended then it would need to be stated more clearly in the rules that this is what is meant.  E.g. line 130 should be extended to explicitly say that the requirement for indirect kicks applies to both the team who has kickoff and the team defending the kickoff.

Reply:
OK. This is up to the TC. Just make it really clear and explicit in the rules (not just by an example) - I think line 130 is the one to modify.

Reply:
I agree that a global rule for both teams is easier for the refs and I am in favor of that. 
However, it seems like the requirement is then saying that no one can score until *both* teams have had a pass. I don't think that is what we want. I think it should be that no one can score until at least one team has two different robots touch the ball.